test/dynamodb/dynamodb.test.js

```diff
@@ -2,37 +2,52 @@ const test = require('ava');
 const AWS = require('aws-sdk');
 const sinon = require('sinon');
 
-const { tableSchema, getConnection, createTables, destroyTables } = require('../src/dynamodb');
+const { tableSchema, getConnection, createTables, destroyTables } = require('../../src/dynamodb');
 
 function throwTestError () {
   throw new Error('test');
 }
 
-test.before(() => {
-  tableSchema([
+const TEST_TABLE_SCHEMA = {
+  TableName: 'test-table',
+  AttributeDefinitions: [
     {
-      TableName: 'test-table',
-      AttributeDefinitions: [
-        {
-          AttributeName: 'hashKey',
-          AttributeType: 'S'
-        },
-        {
-          AttributeName: 'rangeKey',
-          AttributeType: 'S'
-        },
-        {
-          AttributeName: 'gsiHashKey',
-          AttributeType: 'S'
-        },
-        {
-          AttributeName: 'lsiRangeKey',
-          AttributeType: 'S'
-        }
-      ],
+      AttributeName: 'hashKey',
+      AttributeType: 'S'
+    },
+    {
+      AttributeName: 'rangeKey',
+      AttributeType: 'S'
+    },
+    {
+      AttributeName: 'gsiHashKey',
```

Comment:
Why did these attribute names change?

Reply:
The names didn't change, I just pulled it out of the array so I could use it in a later test without having to copy paste it again.

Reply:
Yup, you're right. I thought `hashKey` had change to `gsiHashKe`. I just misread things. Sorry about that.